src/oop/object_independence/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-object_independence-medium-q2
+
+## Question Info
+
+- Question type: Parsons
+
+## Question Text
+
+You've noticed that there are line-ups everywhere. So, you decided to write a program that predicts how long you need to
+wait in them. Make a program called  **PhoneWaitList** that has an attribute called **umAhead**, with a basic accessor
```

Comment:
```suggestion
wait in them. Make a program called  **PhoneWaitList** that has an attribute called **numAhead**, with a basic accessor
```

Reply:
also, everywhere else we call accessor and mutator, getter and setter so for consistency we should probably change it here